blade-graphics/src/lib.rs

```diff
@@ -388,9 +388,9 @@ pub struct TextureSubresources {
 }
 
 #[derive(Debug)]
-pub struct TextureViewDesc<'a> {
+pub struct TextureViewDesc<'a, T> {
```

Comment:
I think we are doing this we should instead just change the signature `create_texture_view` to get the texture as the first argument and the descriptor as the second. This way, the descriptor isn't going to be generic.